exasol/analytics/schema/column.py

```diff
@@ -25,14 +25,19 @@ def decimal_column(
     name: str,
     precision: Optional[int] = None,
     scale: Optional[int] = None,
+    comment: Optional[str] = None,
```

Comment:
This is not used currently

Reply:
Yes, you are right.
Still, I wanted to add an optional comment to some of the entries. 
Alternatively, we could simply insert docstrings into the entries.

Which option do you prefer?

Reply:
See above.

Reply:
add the comments as source code comments

Reply:
see next push